service/history/workflow/mutable_state.go

```diff
@@ -157,6 +157,7 @@ type (
 		DeleteWorkflowTask()
 		DeleteSignalRequested(requestID string)
 		FlushBufferedEvents()
+		GetIncompleteWorkflowExecutionUpdates(context.Context) ([]*historypb.WorkflowExecutionUpdateAcceptedEventAttributes, error)
```

Comment:
Why not:
```suggestion
		GetAcceptedWorkflowExecutionUpdates(context.Context) ([]*historypb.WorkflowExecutionUpdateAcceptedEventAttributes, error)
```
?

Reply:
Sure. I was thinking that if we ever have Admitted entries in here then "Incomplete" is better than "Accepted" for the function name. But then again if that's the case then I'll need to change the return type anyway so it doesn't really save anything.